index.html

```diff
@@ -981,23 +996,32 @@ <h2>
         }
       </pre>
       <p>
-        In this example, two screen wake lock requests are created and released
-        independently:
+        In this example, two different wake lock requests are created and released
+        together using an {{AbortController}}:
       </p>
       <pre class="example js">
-        const lock1 = new WakeLock("screen");
-        const lock2 = new WakeLock("screen");
-
-        lock1.request();
+        const screen = new WakeLock("screen");
+        const system = new WakeLock("system");
 
-        // ...somewhere else
+        const controller = new AbortController();
+        const signal = controller.signal;
 
-        lock2.request();
+        screen.request({ signal });
+        system.request({ signal });
 
-        // ...somewhere else
+        controller.abort();
+      </pre>
+      <p>
+        In this example, all active screen wake locks are being aborted:
+      </p>
+      <pre class="example js">
+        const controller = new AbortController();
```

Comment:
This doesn’t look right? Can you really feed an expired abort controller to things and it cancels everything?

Reply:
Yes, you can - and this allows us to cancel any wake lock, even when created by an imported library etc.

I thought this was a bit weird, and wanted the `abort()` method as that was more clear, but @reillyeon pointed this out and this works too and covers this use-case which might not be super common

Reply:
Ok, but maybe we shouldn't advertise this... this seems kinda bad practice.

Reply:
to be clear, this could lead to all sorts of racy headaches because now now it's unclear who is in control of aborting. I'm not very comfortable with this.

Reply:
Both would be, as the abort steps are added to the signal itself.

Reply:
@domenic or @jakearchibald, would really appreciate some input here. I'm not very familiar with AbortSignal, but this doesn't seems right at all?

Reply:
This is totally fine. I agree it looks weird, but there's no race or other issue here.

Reply:
🤢ok. If this is how they are expected to work, ✅ from me too.

Reply:
@annevk we may want to consider adding a static `AbortSignal.aborted()` factory or similar for this use case.

Reply:
@marcoscaceres it's basically the same as how promises work:

```js
// The promise has already resolved:
const p = Promise.resolve('foo');
// But you can still get the value:
p.then(val => console.log(val));
```

Reply:
I get that bit, sure. But the passing in the aborted signal to things that are holding unaborted signals is the part that seems odd (specially if the newly aborted signals abort the unaborted signals). I’ve not seen promises used that way, thinking off the top of my head.